browser/extensions/BUILD.gn

```diff
@@ -83,6 +77,20 @@ source_set("extensions") {
       "//brave/components/brave_rewards/resources/extension:static_resources",
     ]
   }
+  if (enable_brave_sync) {
+    sources += [
+      "api/brave_sync_api.cc",
+      "api/brave_sync_api.h",
+      "api/brave_sync_event_router.cc",
+      "api/brave_sync_event_router.h",
+    ]
+
+    deps += [
+    "//brave/components/brave_sync",
```

Comment:
nit: - indent

Reply:
addressed in 3b1d457dc14d5a36353c45699459d728c5cb08ef